packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -1,54 +1,64 @@
 import React, { FC, ReactNode } from 'react';
-import classNames from 'classnames';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
 import { Icon } from '../Icon/Icon';
 import { IconName } from '../../types/icon';
 
 export type AlertVariant = 'success' | 'warning' | 'error' | 'info';
 
-interface AlertProps {
+export interface Props {
   title: string;
-  buttonText?: string;
-  onButtonClick?: (event: React.MouseEvent) => void;
   onRemove?: (event: React.MouseEvent) => void;
   severity?: AlertVariant;
   children?: ReactNode;
+  removeButtonContent?: ReactNode | string;
+  /** Deprecated. Use onRemove instead */
+  onButtonClick?: (event: React.MouseEvent) => void;
+  /** Deprecated. Use removeButtonContent instead */
+  buttonText?: string;
 }
 
 function getIconFromSeverity(severity: AlertVariant): string {
   switch (severity) {
-    case 'error': {
-      return 'exclamation-triangle';
-    }
-    case 'warning': {
+    case 'error':
+    case 'warning':
       return 'exclamation-triangle';
-    }
-    case 'info': {
+    case 'info':
       return 'info-circle';
-    }
-    case 'success': {
+    case 'success':
       return 'check';
-    }
     default:
       return '';
   }
 }
 
-export const Alert: FC<AlertProps> = ({ title, buttonText, onButtonClick, onRemove, children, severity = 'error' }) => {
-  const alertClass = classNames('alert', `alert-${severity}`);
+export const Alert: FC<Props> = ({
+  title,
+  buttonText,
+  onButtonClick,
+  onRemove,
+  children,
+  removeButtonContent,
+  severity = 'error',
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme, severity, !!removeButtonContent);
+
   return (
-    <div className="alert-container">
-      <div className={alertClass}>
-        <div className="alert-icon">
+    <div className={styles.container}>
+      <div className={styles.alert} aria-label={`alert-${severity}`}>
```

Comment:
Think we can define this label in the e2e-selectors package and use it both here and in tests.